neo/rawio/biocamrawio.py

```diff
@@ -302,5 +325,120 @@ def readHDF5t_101_i(rf, t0, t1, nch):
 
 def readHDF5t_brw4(rf, t0, t1, nch):
     for key in rf:
-        if key[:5] == "Well_":
+        if key.startswith("Well_"):
             return rf[key]["Raw"][nch * t0 : nch * t1]
+
+
+def readHDF5t_brw4_sparse(rf, t0, t1, nch, true_zeroes=False, use_synthetic_noise=False):
+
+    # noise_std = None
+    start_frame = t0
+    num_frames = t1 - t0
+    for well_ID in rf:
+        if well_ID.startswith("Well_"):
+            break
+    # initialize an empty (fill with zeros) data collection
+    data = np.zeros((nch, num_frames), dtype=np.int16)
```

Comment:
@mahlzahn @b-grimaud another quick question: shouldn't this be `unit16`? And the `int.from_bytes` should be `signed=False`?

Reply:
Yes, because the stream is defined as uint..but I think it's inconsistent and it might lead to problema later